plugins/backstage-plugin-coder/src/api/api.ts

```diff
@@ -49,34 +46,15 @@ async function getCoderApiRequestInit(
   };
 }
 
-// Makes it easier to expose HTTP responses in the event of errors and also
-// gives TypeScript a faster way to type-narrow on those errors
-export class BackstageHttpError extends Error {
-  #response: Response;
-
-  constructor(errorMessage: string, response: Response) {
-    super(errorMessage);
-    this.name = 'HttpError';
-    this.#response = response;
-  }
-
-  get status() {
-    return this.#response.status;
-  }
-
-  get ok() {
-    return this.#response.ok;
-  }
-
-  get contentType() {
-    return this.#response.headers.get('content_type');
-  }
-}
+type TempPublicUrlSyncApi = Readonly<{
+  getApiEndpoint: UrlSync['getApiEndpoint'];
+  getAssetsEndpoint: UrlSync['getAssetsEndpoint'];
+}>;
```

Comment:
Hoping to get rid of this in the next PR. I just couldn't without making the `CoderClient` work start bleeding into things

Reply:
Removed as of the next PR